test/Sentry.Tests/TransactionProcessorTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Sentry.Tests;
+
+[UsesVerify]
+public class TransactionProcessorTests
```

Comment:
Need some more tests:

- Discarding a transaction in a transaction processor works as expected
- Discarding a transaction in a transaction processor logs the correct client report reason and category
- A transaction that is sampled out is not run through a transaction processor

Reply:
👍🏻